test/jdk/java/lang/invoke/VarHandles/generate-vh-tests.sh

```diff
@@ -41,6 +41,12 @@ do
       ;;
   esac
 
+  case $type in
+    Point)
+      args="$args -KPrimitiveClass"
```

Comment:
I don't think PrimitiveClass is needed.

```
  case $type in
    boolean|byte|short|char|int|long|float|double|Point)
      args="$args -KValue"
      ;;
  esac
```  

"Value" is used to annotate as primitive class because "value class" was the terminology when these tests were updated.

You can rename `Value` to `PrimitiveClass` in this file and `X-VarHandle*` templates.  Or leave it as is and we will change that when we add tests for value objects.